src/main/java/net/minecraftforge/event/AttachCapabilitiesEvent.java

```diff
@@ -133,4 +133,21 @@ public Item(net.minecraft.item.Item item, net.minecraft.item.ItemStack stack)
             return this.stack;
         }
     }
+
+    /**
+     * A version of the parent event which is only fired for Worlds.
+     */
+    public static class World extends AttachCapabilitiesEvent
+    {
+        private final net.minecraft.world.World world;
+        public World(net.minecraft.world.World world)
```

Comment:
Because the sub class itself is called "World", meaning just World would be a self reference